brands/shop/car.json

```diff
@@ -253,12 +253,12 @@
       "shop": "car"
     }
   },
-  "shop/car|Skoda": {
+  "shop/car|Ŝkoda": {
```

Comment:
@sb12 I did a short research and every source that I found (official website of the company in multiple languages and Wikipedia) are suggesting that the right spelling would be `Škoda` instead of `Ŝkoda`. Are you really sure, that the current spelling is correct?

Reply:
Of course, you are right. 
As I am not familiar with Czech language I didn't notice the difference between Škoda and Ŝkoda.
Thanks for spotting it!